plugins/modules/ovirt_auth.py

```diff
@@ -83,6 +94,7 @@
                Note that this is a hint for the server, and that it may return
                uncompressed data even when this parameter is set to I(True)."
         type: bool
+        default: true
```

Comment:
Do we have any clear guideline around cases for boolean values (for example `true` vs `True`)? It seems to me we have mixed usage there ...